package.json

```diff
@@ -104,15 +104,15 @@
 	},
 	"jest": {
 		"collectCoverageFrom": [
-			"(blocks|components|date|editor|element|i18n|data|utils)/**/*.js"
```

Comment:
We also include a custom ESLint rule for each top-level directory in `.eslintrc.json`:

https://github.com/WordPress/gutenberg/blob/e694f6a99b7ce01bb3586798e1af6c662a7a4895/.eslintrc.json#L95-L125